src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -417,7 +417,7 @@ void DlgPrefLibrary::slotSeratoMetadataExportClicked(bool checked) {
                             "recommended. Are you sure you want to enable this "
                             "option?"),
                     QMessageBox::Yes | QMessageBox::No) == QMessageBox::No) {
-            checkBox_SeratoMetadataExport->setChecked(false);
+            checkBox_SeratoMetadataExport->setChecked(true);
```

Comment:
You changed the default state and inverted the state if the user explicitely selected that option.
This is not WIP, this doesn't make any sense at all.

The title of #11226 is "Write Serato Metadata checkbox is ignored when writing track metadata is disabled"
= the state of the Serato checkbox is ignored if the checkbox above ("Write track metadata...") is unchecked
Fix: the GUI should reflect the Serato state is irrelevant
= disable (grey out) the Serato checkbox in that case

Reply:
Ahhh, got it I was thinking the other way around.

Reply:
I should close this PR and open a new one instead.